package.json

```diff
@@ -121,18 +122,18 @@
     "globals": "^16.1.0",
     "history": "^5.3.0",
     "html-webpack-plugin": "^5.6.3",
-    "jest": "^28.1.3",
-    "jest-axe": "^6.0.0",
-    "jest-environment-jsdom": "^30.0.0-alpha.3",
-    "jest-resolve": "^28.1.3",
-    "jest-watch-typeahead": "^2.0.0",
-    "npm-run-all2": "^6.0.6",
+    "jest": "^29.7.0",
+    "jest-axe": "^10.0.0",
+    "jest-environment-jsdom": "^30.0.0-alpha.7",
+    "jest-resolve": "^29.7.0",
+    "jest-watch-typeahead": "^2.2.2",
```

Comment:
I personally don't use this extension to jest, but I still kept it to be sure...